lib/screens/choose_game_screen.dart

```diff
@@ -129,7 +148,7 @@ class _ChooseGameScreenState extends State<ChooseGameScreen> {
               ),
               child: ElevatedButton(
                 onPressed: () {},
-                style: theme.elevatedButtonTheme.style!.copyWith(
+                style: theme.elevatedButtonTheme.style?.copyWith(
```

Comment:
```suggestion
                style: theme.elevatedButtonTheme.style!.copyWith(
```
As long as its defined in the consructor of `MaterialApp`, doing non-null assertions should be fine.